config/locales/en.yml

```diff
@@ -398,7 +398,33 @@ en:
       title: Settings
       general:
         title: General
+        heading: Project name and description
         submit: Update project
+      advanced:
+        title: Advanced settings
+        path:
+          title: Change path
+          description: A project's path is a unique identifier where it can be accessed in the URL
```

Comment:
Can you update this text to read `A project's path is a unique identifier that can be accessed via the URL`

Reply:
Sure, updated in 52534af Thanks

Reply:
Awesome thanks!